sc_server_auth/server/models.py

```diff
@@ -5,23 +5,26 @@
 from pydantic import BaseModel
 from pydantic.class_validators import validator
 
-from sc_server_auth.config import params
-from sc_server_auth.server import constants as cnt
+from sc_server_auth.configs.models import Messages
+from sc_server_auth.configs.parser import get_config
+from sc_server_auth.configs.paths import PUBLIC_KEY_PATH
+
+config = get_config().tokens
 
 
 def _validate_token(value):
     try:
-        with open(params[cnt.PUBLIC_KEY_PATH], "rb") as file:
+        with open(PUBLIC_KEY_PATH, "rb") as file:
             public_key = file.read()
-            jwt.decode(value, public_key, issuer=params[cnt.ISSUER], algorithm="RS256")
+            jwt.decode(value, public_key, issuer=config.issuer, algorithm="RS256")
     except (
         jwt.exceptions.InvalidTokenError,
         jwt.exceptions.InvalidSignatureError,
         jwt.exceptions.InvalidIssuerError,
         jwt.exceptions.ExpiredSignatureError,
         FileNotFoundError,
```

Comment:
Should auth-service start if file with key wasn't found? Is it good idea to check an existence of the public key before starting service?

Reply:
Yes, good idea